Resources/public/js/views/ez-actionbarview.js

```diff
@@ -59,7 +59,7 @@ YUI.add('ez-actionbarview', function (Y) {
                         }),
                         new Y.eZ.CreateContentActionView({
                             actionId: 'createContent',
-                            disabled: false,
+                            disabled: !this.get('contentType').get('isContainer'),
```

Comment:
I would move this logic to the CreateContentActionView meaning it should receive the contentType and set its `disabled` flag depending on it.